crates/re_types_builder/src/format/python.rs

```diff
@@ -78,16 +81,9 @@ fn format_path_for_tmp_dir(
 fn format_python_dir(dir: &Utf8PathBuf) -> anyhow::Result<()> {
     re_tracing::profile_function!();
 
-    // The order below is important and sadly we need to call black twice. Ruff does not yet
-    // fix line-length (See: https://github.com/astral-sh/ruff/issues/1904).
-    //
-    // 1) Call black, which among others things fixes line-length
-    // 2) Call ruff, which requires line-lengths to be correct
-    // 3) Call black again to cleanup some whitespace issues ruff might introduce
-
-    run_black_on_dir(dir).context("black")?;
-    run_ruff_on_dir(dir).context("ruff")?;
-    run_black_on_dir(dir).context("black")?;
```

Comment:
This was introduced in https://github.com/rerun-io/rerun/pull/2686

I'm not sure this is needed anymore, since the above PR didn't give a repro. Let's hope not?

Reply:
It was happening deterministically from codegen-generated code at the time.  We could check out the parent commit if we really want a repro.

As long as `ruff --format` is a no-op after running `cargo codegen` then I think we're good.

Reply:
Great, then we are good!